src/main/resources/fabric.mod.json

```diff
@@ -29,8 +29,8 @@
     "additionalentityattributes.mixins.json"
   ],
   "depends": {
-    "fabricloader": "*",
+    "fabricloader": "0.15",
```

Comment:
This will break when loader updates, should be `>=0.15`, imo.
I can tweak this when porting jump height, so no problem.

Reply:
this was a mistake, thanks for fixing it.